src/common/line_entry.rs

```diff
@@ -50,6 +50,13 @@ impl LineEntry {
         self.raw_string.trim()
     }
 
+    pub fn stripped_export_string(&self) -> &str {
```

Comment:
```suggestion
   fn stripped_export_string(&self) -> &str {
```
This function shouldn't be public.